lib/vscode/.eslintrc.json

```diff
@@ -88,7 +88,7 @@
 			}
 		],
 		"code-import-patterns": [
-			"warn",
+			"off",
```

Comment:
What was the reason for turning these off?

Reply:
I believe it was off before: see [here](https://github.com/cdr/code-server/blob/fc6d123da59a4e5a675ac8e080f66e032ba01a1b/lib/vscode/.eslintrc.json#L92-L93)

I messed up and turned it to `warn` in the rebase but wasn't sure how to revert that specific change.

Reply:
Ah I gotchu. I wonder why we turned them off. I might experiment turning them back on and see what happens.

Reply:
I experimented and it causes issues since we do some importing across "layers" (browser, common, node, electron) that are technically not allowed (but work fine).

So we might want to refactor a bit later so we can turn the checks on but it's probably not a big deal.